### PR TITLE
Added Azure Profiles description

### PR DESCRIPTION
**Pull request by dcaro**
## Description

Adding a description for Azure Profiles

## Checklist

- [X] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: documentation/preview/AzureProfiles.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+```
+
+Only the following names four profiles are supported in the current preview:
+
+```PowerShell
+hybrid-2019-03-01
+latest-2019-04-30
+```
+
+If enter a value that is not supported, the command will not return an error but will not switch profiles.
```
Comment (sphibbs) [inline documentation/preview/AzureProfiles.md]: Known issue. Or some other text to highlight there is badness beware.

File: documentation/preview/AzureProfiles.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+We will be adding profiles over time and we plan to phase out older profiles after a certain time.
+
+## Usage
+
+```PowerShell
+Select-AzProfile -name 'hybrid-2019-03-01'
+```
+
+Only the following names four profiles are supported in the current preview:
```
Comment (sphibbs) [inline documentation/preview/AzureProfiles.md]:
```suggestion
Only the following profiles are supported in the current preview:
```

**Comment by dcaro** [comment]
@sphibbs thanks, added a clearer call on the behavior

**Pull request merged**